frontend/public/locales/ko/public.json

```diff
@@ -489,13 +489,13 @@
   "Unresolved": "미해결",
   "There is no image associated with this tag": "이 태그와 관련된 이미지가 없습니다.",
   "Image registry commands": "이미지 레지스트리 명령",
-  "Create a new Image Stream Tag by pushing an image to this Image Stream with the desired tag.": "원하는 태그를 사용하여 이미지를이 이미지 스트림에 푸시하여 새 이미지 스트림 태그를 만듭니다.",
+  "Create a new ImageStream Tag by pushing an image to this ImageStream with the desired tag.": "원하는 태그를 사용하여 이미지를이 이미지 스트림에 푸시하여 새 이미지 스트림 태그를 만듭니다.",
```

Comment:
We're going to do one more file exchange with the translation team before code freeze, so there's no need to manually update the files in ko, zh, and ja.

Reply:
I removed the changes to the ko, zh, and ja public.json files